backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -407,22 +407,19 @@ public Workspace getWorkspaceByWorkspaceName(
    * workspace.
    */
   @Override
-  public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
+  public WorkspaceDTO getWorkspaceDTOByWorkspaceIdForServiceUser(
       UserInfo currentLoginUserInfo, String workspaceId, Integer workspaceType) {
     WorkspaceDTO workspaceDTO = new WorkspaceDTO();
     workspaceDTO.setWorkspaceId(workspaceId);
 
     switch (workspaceType) {
       case WorkspaceType.ORGANIZATION_VALUE:
-        Organization organization = (Organization) getOrgById(workspaceId);
+        Organization organization = (Organization) getOrgById(true, workspaceId, true);
         workspaceDTO.setWorkspaceType(WorkspaceType.ORGANIZATION);
         workspaceDTO.setWorkspaceName(organization.getName());
         return workspaceDTO;
       case WorkspaceType.USER_VALUE:
         workspaceDTO.setWorkspaceType(WorkspaceType.USER);
-        if (currentLoginUserInfo == null) {
-          currentLoginUserInfo = authService.getCurrentLoginUserInfo();
-        }
```

Comment:
?

Reply:
This method is no more called by any user and it will only called by service user from background so not need to fetch `getCurrentLoginUserInfo` so removed it.